src/modules/robot/Robot.cpp

```diff
@@ -1555,7 +1558,7 @@ void Robot::reset_position_from_current_actuator_position()
 // Convert target (in machine coordinates) to machine_position, then convert to actuator position and append this to the planner
 // target is in machine coordinates without the compensation transform, however we save a compensated_machine_position that includes
 // all transforms and is what we actually convert to actuator positions
-bool Robot::append_milestone(const float target[], float rate_mm_s, unsigned int line)
+bool Robot::append_milestone(const float target[], float feed_rate, unsigned int line)
```

Comment:
this now accepts the feed rate as it was provided in the G-code, instead of mm/sec. I've updated all the call sites I could find, please let me know if I've missed something.